crates/typst/src/layout/place.rs

```diff
@@ -27,10 +39,6 @@ pub struct PlaceElem {
     ///
     /// - If `float` is `{false}`, then this can be any alignment other than `{auto}`.
     /// - If `float` is `{true}`, then this must be `{auto}`, `{top}`, or `{bottom}`.
-    ///
-    /// When an axis of the page is `{auto}` sized, all alignments relative to
-    /// that axis will be ignored, instead, the item will be placed in the
-    /// origin of the axis.
```

Comment:
Any reason for this removal? Is this information outdated? Or is it about the "an axis of the page" bit? We could perhaps replace it with "an axis of the parent container" or similar.

Reply:
I guess it's outdated because I cannot reproduce anything like that. For example the following respects the requested alignment for both axes, although they are both `auto`-sized:

```
#set page(width: auto, height: auto)
#rect(width: 8cm, height: 1cm)
#place(horizon+center, [x])
```

By the way I just added another sentence here instead, to mention that the `y` position defaults to the current position if no vertical alignment is specified.